cli-tests/src/upload.rs

```diff
@@ -205,6 +206,37 @@ async fn upload_bundle_no_files() {
     println!("{assert}");
 }
 
+#[tokio::test(flavor = "multi_thread")]
```

Comment:
Do you know why this is necessary / used, besides the fact that the other tests do it here? I don't either, but I wanted to ask

Reply:
I don't know, I am cargo culting 😱